modules/margin/src/main/java/com/opengamma/sdk/margin/Ccp.java

```diff
@@ -17,20 +17,46 @@
  */
 public class Ccp implements Comparable<Ccp>, Serializable {
 
+  /** ASX (Australian Securities Exchange) SPAN. */
+  public static final Ccp ASX_SPAN = new Ccp("ASX_SPAN");
```

Comment:
Really, our external margin API should not use underscores. We should map them on the server side.